### PR TITLE
fix(web): supporting views

### PR DESCRIPTION
**Pull request by coolguy741**
# Overview
refactoring initialization part of views.
fixed navigating newly created view

### CODE REVIEW

File: web/src/components/organisms/Project/Content/ViewsMenu/hooks.ts [hunk 1]
```diff
@@ -41,20 +41,16 @@ export default ({ modelId, currentView, setCurrentView }: Params) => {
   });
 
   const views = useMemo(() => {
-    if (!loading && data?.view) {
+    if (loading && data?.view) {
```
Comment (nourbalaha) [inline web/src/components/organisms/Project/Content/ViewsMenu/hooks.ts:44]: Why did you remove the exclamation mark? this is causing the initial view to not be loaded correctly.

**Review by nourbalaha — CHANGES_REQUESTED** [review]

File: web/src/components/organisms/Project/Content/ViewsMenu/hooks.ts (hunk 1 above)
Comment (coolguy741) [reply web/src/components/organisms/Project/Content/ViewsMenu/hooks.ts:44]: With exclamation mark, navigating to newly created view doesn't work.

Comment (nourbalaha) [reply web/src/components/organisms/Project/Content/ViewsMenu/hooks.ts:44]: initial view and newly created view should be working at the same time

Comment (caichi-t) [reply web/src/components/organisms/Project/Content/ViewsMenu/hooks.ts:44]: I think he did it for navigating to newly created view but it cause the issue so we should do other solution.

Comment (coolguy741) [reply web/src/components/organisms/Project/Content/ViewsMenu/hooks.ts]: Sorry. I was wrong. I created prevModelId state.

**Review by nourbalaha — APPROVED** [review]
